### PR TITLE
Minor doc updates, mostly about `jj workspace`

### PR DESCRIPTION
**Pull request by martinvonz**
# Checklist

If applicable:
- [ ] I have updated `CHANGELOG.md`
- [ ] I have updated the documentation (README.md, docs/, demos/)

### CODE REVIEW

File: docs/working-copy.md [hunk 1]
```diff
@@ -80,3 +77,19 @@ See https://git-scm.com/docs/gitignore for details about the format.
 `.gitignore` files are supported in any directory in the working copy, as well
 as in `$HOME/.gitignore`. However, `$GIT_DIR/info/exclude` or equivalent way
 (maybe `.jj/gitignore`) of specifying per-clone ignores is not yet supported.
+
+
+## Workspaces
```
Comment (arxanas) [inline docs/working-copy.md:82]: Is this the equivalent of git worktrees?

File: docs/working-copy.md [hunk 2]
```diff
@@ -80,3 +77,19 @@ See https://git-scm.com/docs/gitignore for details about the format.
...
+You can have multiple working copies backed by a single repo. Use 
+`jj workspace add` to create a new working copy. The working copy will have a
+`.jj/` directory linked to the main repo. The working copy and the `.jj/`
+directory together is called a "workspace". Each workspace can have a different
+commit checked out.
+
+Having multiple workspaces can be useful for running long-running tests in a one
+while you continue developing in another, for example.
+
+When you're done using a workspace, use `jj workspace forget` to make the repo
```
Comment (arxanas) [inline docs/working-copy.md:93]: Does anything need to be done if either the main repo or the child repo is moved on disk?

**Review by arxanas — APPROVED** [review]

File: docs/working-copy.md (hunk 2 above)
Comment (martinvonz) [reply docs/working-copy.md:93]: > Does anything need to be done if either the main repo or the child repo is moved on disk?

Yes, you need to move them back :) And yes, that obviously needs to be improved. I know Git has `git worktree repair`.

File: docs/working-copy.md (hunk 1 above)
Comment (martinvonz) [reply docs/working-copy.md:82]: Yes

File: docs/working-copy.md (hunk 2 above)
Comment (martinvonz) [reply docs/working-copy.md:93]: Correction: You only need up move the main workspace back; the repo (which lives in the main workspace) doesn't know the location of the other workspaces. The repo has an ID for each workspace (including its own) and each workspace records its own ID.

**Pull request merged**